xmtp_mls/src/groups/sync.rs

```diff
@@ -77,6 +78,13 @@ use xmtp_proto::xmtp::mls::{
     },
 };
 
+#[derive(Debug)]
+struct PublishIntentData {
+    staged_commit: Option<Vec<u8>>,
+    post_commit_action: Option<Vec<u8>>,
+    message: Vec<u8>,
```

Comment:
I wonder if `payload` or `payload_to_publish` would be a clearer name here, as `message` could be confused with application message, and it also doesn't convey what stage of encryption/marshaling it is in (from looking at the rest of the code, I think it's fully encrypted and marshaled here)

Reply:
That's a good call. Easy enough to change.